actions/bcr-pr-review-approver/index.js

```diff
@@ -0,0 +1,262 @@
+const { getInput, setFailed } = require('@actions/core');
+const { context, getOctokit } = require("@actions/github");
+
+// Keep this function in sync with the one in actions/bcr-pr-review-notifier/index.js
+async function fetchAllModifiedModules(octokit, owner, repo, prNumber) {
```

Comment:
how annoying is it to share this code? (I don't know how nodejs `require` works, so if it's very annoying, feel free to leave as is -- or maybe add the same comment to the other file)

Reply:
I didn't find a easy way to do it, we might have to create a third GitHub Action to do so. I'll keep it as it is for now and add comment in the other file. Hopefully this code will mostly stay stable.